java/core/src/test/scala/com/azavea/gdal/LockedDatasetSpec.scala

```diff
@@ -0,0 +1,10 @@
+package com.azavea.gdal
+
+import org.scalatest._
+
+class LockedDatasetSpec extends FunSpec with Matchers with BeforeAndAfterAll {
+  it("Test that dylibs are laoded") {
+    LockedDataset
```

Comment:
Just a test example to show that a dynamic lib generated by the `native` project is in the scope if this `core` project. Should be pretty convenient to develop.